### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by turnerm**
Addresses #180

### CODE REVIEW

File: src/ochanticipy/datasources/codab/codab.py [hunk 1]
```diff
@@ -71,13 +71,13 @@ def process(self, *args, **kwargs):
...
 
-    def load(self, admin_level: int) -> gpd.GeoDataFrame:  # type: ignore
+    def load(self, admin_level: int = 0) -> gpd.GeoDataFrame:  # type: ignore
         """
         Get the COD AB data by admin level.
 
         Parameters
         ----------
-        admin_level: int
+        admin_level: int, default = 0
```
Comment (caldwellst) [inline src/ochanticipy/datasources/codab/codab.py:80]: Nice. I made changes in the `feature/add_hrp_countries` branch but have changed to match this so we don't have to resolve later.

File: src/ochanticipy/datasources/glofas/glofas.py [hunk 2]
```diff
@@ -138,6 +139,15 @@ def __init__(
             dtstart=self._start_date,
             until=self._end_date,
         )
+        if self._date_range.count() > _CDS_MAX_REQUESTS:
```
Comment (caldwellst) [inline src/ochanticipy/datasources/glofas/glofas.py:142]: So, question, could an eventual implementation make the maximum requests and then sleep before making the additional ones, preventing the user from having to create two separate instances if they want to analyse data across a time that contains more points than `_CDS_MAX_REQUESTS`?

Comment (caldwellst) [reply src/ochanticipy/datasources/glofas/glofas.py:142]: Not saying to be done now, but should it be an issue or no?

File: src/ochanticipy/utils/check_file_existence.py [hunk 3]
```diff
@@ -65,7 +65,7 @@ def check_file_existence(
     # check filepath exists -> clobber
     usage_dict = {
         True: {True: "overwriting existing", False: "using existing"},
-        False: {True: "downloading new", False: "downloading new"},
```
Comment (caldwellst) [inline src/ochanticipy/utils/check_file_existence.py:68]: Good catch.

**Review by caldwellst — APPROVED** [review]
All looks good to me! Just a couple of questions added, but nothing critical to merging

File: src/ochanticipy/datasources/glofas/glofas.py (hunk 2 above)
Comment (turnerm) [reply src/ochanticipy/datasources/glofas/glofas.py:142]: Oh yeah definitely, good question. It's actually how I started writing this module in the first place, using `asyncio` and a producer / consumer architecture. Then I realized that the request limit was very high and we wouldn't reach it (at the time, this was a few months ago...) so I dropped this plan, as `asyncio` makes things more complicated and testing hard. 

But _now_ with the forecast we do reach the queue limit if asking for all historical available forecasts, oops. So indeed this is a temporary fix.

Comment (turnerm) [reply src/ochanticipy/datasources/glofas/glofas.py:142]: Ah yes  and it should be an issue, let me add it

**Pull request merged**